content/posts/about_inlining.md

```diff
@@ -9,11 +9,11 @@ tags = [
 
 ## Introduction
 
-For me, everytime I encounter the stituation and document myself about (chỗ này hơi bị bế quất, đọc lại vài lần rồi nói cho tao ý của mày nhé), I still end up with the same conclusions:
+For me, every time I encounter the `inline` keyword in codebase and document myself about it, I still end up with the same conclusions:
 
 > Inlining is often to be misused in development, ending up in counter-productive and risky situations.
 
-I will try to summarize all the things which I believe we need to remember about inlining, giving pros, cons and situations to use it and to not use it.
+I will try to summarize all the things which I believe we need to remember about inlining, giving pros, cons, and situations to use it and to not use it.
```

Comment:
[Refactor] "All the things" should be "all things".
[Suggestion] If you don't want to use "things", you should specific a noun such as aspect or crteria, or any others with such meaning.
[Suggestion] "situations to use it and to not use it" should be "situations when it can be used and when we should avoid"
[Suggestion] "giving pros and cons when using it. I also provide some situations for you to consider to use or not."